lib/erb_lint/runner.rb

```diff
@@ -3,8 +3,9 @@
 module ERBLint
   # Runs all enabled linters against an html.erb file.
   class Runner
-    def initialize(config)
-      @config = config
+    def initialize(config = {})
+      @config = default_config.merge(config || {})
```

Comment:
Maybe it's too defensive but that means that the styleguide needs to be changed cause it's currently passing a `nil` config when it can't properly parse the YAML.